release/release-notes/odfe-data-prepper.release-notes-1.0.0.md

```diff
@@ -0,0 +1,25 @@
+# 2021-05-xx Version 1.0.0
+
+## Highlights
+* Data Prepper now builds using version 1.0+ of the OpenTelemetry tracing specification
+* Additional TraceGroup fields are emitted for enhanced searching and filtering with the Trace Analytics Kibana plugin
+* An OpenSearch-compatible version of Data Prepper has been released under the [opensearch-project repo](https://github.com/opensearch-project/data-prepper/releases)
+
+## Compatibility
+### Issue #1
+This release is intended to be used by version 1.13.2.1+ of the Trace Analytics Kibana plugin. The most recent Open Distro for Elasticsearch bundle contains version 1.13.2.0, so only use this version of Data Prepper if you have manually upgraded the Trace Analytics plugin.
```

Comment:
This looks fine to me as long as we confirm the upcoming of 1.13.2.1+.

Reply:
Good point - will wait for feedback. If we aren't sure of a definitive release then I can update to say that the TA plugin has to be built from source for it to work with DP 1.0.0.